modules/@apostrophecms/page-type/index.js

```diff
@@ -392,9 +392,6 @@ module.exports = {
   },
   extendMethods(self) {
     return {
-      enableAction() {
```

Comment:
Why the change back?

Reply:
(Changing this back is not a blocker for me.)

Reply:
I wasn't sure we were settled on this approach. I [suggested coming at this from the other direction here](https://github.com/apostrophecms/apostrophe/pull/2928#issuecomment-823262175), but if we like inheriting `page`'s action I'll push ahead in the other PR with the suggested cleanup

Reply:
Sure, a separate PR would be fine.